content/faq/libraries-frameworks/word-excel.mdx

```diff
@@ -9,3 +9,5 @@ showToc: false
 - [Close XML](https://github.com/closedxml/closedxml) .NET library for reading, manipulating and writing Excel 2007+ (.xlsx, .xlsm) files.
 - [Epplus](https://github.com/EPPlusSoftware/EPPlus) Excel spreadsheets for .NET
 - [NPOI](https://github.com/nissl-lab/npoi) .NET version of POI Java project. With NPOI, you can read/write Office 2003/2007 files very easily.
+- [DevExpress Workbook] (https://docs.devexpress.com/OfficeFileAPI/14912/spreadsheet-document-api)(not free) easily read/write data to dataTable 
```

Comment:
Could you please add short description what is data table? I am not sure it is data table in ADO.net or something else.